raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -190,7 +190,34 @@ library NettingChannelLibrary {
         self.closed = block.number;
     }
 
-    /// @notice close(bytes, bytes) to close a channel between to parties
+
+    /// @notice close a channel between two parties without a raiden transfer
+    ///         Cases where this may need to happen is:
+    ///         - Alice deposits but Bob doesn't, then Alice must be able to
+    ///           close the channel and reclaim her deposit.
+    ///         - Neither Alice or Bob have made any deposits
+    ///         - Generally any case where you want to simply close the channel
+    ///           without adding an extra transfer to it
+    /// @param caller_address The address of the participant trying to close
```

Comment:
We should probably update these comments to reflect the change of semantics with the raiden channels.

Initially all raiden channels were funded by both parties, after some time we realized that it's not a requirement that both parties deposit to make the channel useful, ie. we can have single funded channels. This function is actually best described as a way to close a channel that was never used (or at least that is what the closing address is saying).